crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -286,6 +290,41 @@ public enum DefaultClippingState
 
         BuildCommandBuffer _commandbufferBuilder;
 
+        public struct CascadeParams
+        {
+            public Vector2 _posSnapped;
+            public float _scale;
+
+            public float _textureRes;
+            public float _oneOverTextureRes;
+
+            public float _texelWidth;
+
+            public float _weight;
+
+            // Align to 32 bytes
+            public float __padding;
+        }
+        public ComputeBuffer _bufCascadeDataTgt;
+        public ComputeBuffer _bufCascadeDataSrc;
+
+        public struct PerCascadeInstanceData
+        {
+            public float _meshScaleLerp;
+            public float _farNormalsWeight;
+            public float _geoGridWidth;
+            public Vector2 _normalScrollSpeeds;
+
+            // Align to 32 bytes
+            public Vector3 __padding;
+        }
+        public ComputeBuffer _bufPerCascadeInstanceData;
+
+        CascadeParams[] _cascadeParamsSrc = new CascadeParams[LodDataMgr.MAX_LOD_COUNT + 1];
```

Comment:
Minor point, but something that might be worth breaking-out into a wider discussion. Now that we are on 2019.4 it might be better to default to using `NativeArray<T>` instead of C# managed arrays. Unity recommend using the former where possible because even if you don't use burst they can have slightly better perf characteristics - and if we do ever decide to burstify our code - this would make doing so easier.

Reply:
I wont manage to do it for this PR if thats ok but i'm fine with it in principle if we're ok diverging further from 2018 which we probably are?

Reply:
Sounds like a good idea.